src/components/datetime.js

```diff
@@ -13,6 +13,11 @@ module.exports = function(app) {
         controller: ['$scope', '$timeout', function($scope, $timeout) {
           if ($scope.builder) return;
 
+          // Close calendar pop up when tabbing off button
+          $scope.onKeyDown = function(event) {
```

Comment:
Could we not accomplish this with a `blur` event?

https://www.w3schools.com/jsref/event_onblur.asp